mri/parallel_mri/extract_sensitivity_maps.py

```diff
@@ -121,17 +126,16 @@ def gridded_inverse_fourier_transform_stack(kspace_plane_loc, z_sample_loc,
     np.ndarray
         The gridded inverse fourier transform of given kspace data
     """
-    gridded_kspace = []
+    gridded_kspace = np.zeros(shape)
     stack_len = len(kspace_plane_loc)
-    for i in range(len(z_sample_loc)):
-        gridded_kspace.append(
-            griddata(kspace_plane_loc,
-                     kspace_data[i*stack_len:(i+1)*stack_len],
-                     grid,
-                     method=method,
-                     fill_value=0))
-    # Move the slice axis to last : Make to Nx x Ny x Nz
-    gridded_kspace = np.moveaxis(np.asarray(gridded_kspace), 0, 2)
+    for i, idx_z in enumerate(idx_mask_z):
+        gridded_kspace[:, :, idx_z] = griddata(
+            kspace_plane_loc,
+            kspace_data_sorted[i*stack_len:(i+1)*stack_len],
+            grid,
+            method=method,
+            fill_value=0)
```

Comment:
nitpicking here, but to handle multi-line in the best way possible you can even do:
```python
gridded_kspace[:, :, idx_z] = griddata(
  kspace_plane_loc,
  kspace_data_sorted[i*stack_len:(i+1)*stack_len],
  grid,  
  method=method,
  fill_value=0,
)
```
If for example someone wants to add an argument at the end, they don't need to modify the `fill_value` line too.